src/sublime_aio.py

```diff
@@ -87,10 +105,18 @@ async def on_modified(self):
             print("debounced ViewEventListener.on_modified_async", self.view.id())
     ```
     """
-    def decorator(coro_func: Coroutine) -> Callable:
-        call_at = {}
+    @overload
+    def decorator(coro_func: Callable[[EL, sublime.View], BlankCoro]) -> Callable[[EL, sublime.View], None]: ...
```

Comment:
Not sure how clear it is for endusers `EL` pointing to `sublime_aio.EventListener` and `VEL` to `sublime_aio.ViewEventListener` assuming they take the time to analyze and understand the messy output at all.

![grafik](https://github.com/user-attachments/assets/c698f049-b16e-481d-b984-829705f76ed3)

Reply:
Unfortunately, we need these typevars.

This is with the typevars replaced with type aliases for the actual class types:

![image](https://github.com/user-attachments/assets/020cecd8-39cd-40a2-95ee-21408c6a4afc)